.github/dependabot.yml

```diff
@@ -0,0 +1,17 @@
+# Until bug below is sorted we will not allow dependabot to run by itself for npm dependencies
+# https://github.com/dependabot/dependabot-core/issues/1190
```

Comment:
Do we need this comment for this repository? :) 
```suggestion
```

Reply:
ah yeah indeed not needed here 😅

Reply:
thanks :)